Packs/CrowdStrikeFalconSandbox/Playbooks/playbook-Detonate_URL_-_CrowdStrike.yml

```diff
@@ -247,22 +275,67 @@ tasks:
       iscommand: false
       exitCondition: ""
       wait: 1
+      max: 0
     view: |-
       {
         "position": {
           "x": 162.5,
-          "y": 740
+          "y": 895
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "17":
+    id: "17"
+    taskid: 063f8872-26bf-4bf9-8caa-645a82166659
+    type: condition
+    task:
+      id: 063f8872-26bf-4bf9-8caa-645a82166659
+      version: -1
+      name: Is url detonated?
```

Comment:
why is this needed?

Reply:
If the url did not detonated - Skip to the done task and not fail the playbook

Reply:
change to `Was job created ? (could be it wasn't created because file type is not supported)`